src/apps/material-search/MaterialSearchInputs.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from "react";
+import { useText } from "../../core/utils/text";
+import { Work } from "../../core/utils/types/entities";
+
+interface MaterialSearchInputsProps {
+  searchInput: string;
+  setSearchInput: (value: string) => void;
+  availableMaterialTypes: string[] | null;
+  selectedMaterialType: string | null;
```

Comment:
I guess yes. I used string because i understood it like we don't know what types will come from the data, and they can be changed from their side. 

I have changed it throughout now to use `ManifestationMaterialType`, as i can see has been done elsewhere, but it feels weird to me that we would do this instead of string, since we aren't generating these anyway.